apps/total-typescript/src/pages/[article].tsx

```diff
@@ -40,6 +41,16 @@ export const getStaticProps: GetStaticProps = async ({params}) => {
       },
     }))
 
+  const shortenedArticleBody =
```

Comment:
```tsx
let articleBody
if (article.withEmailWall) {
  articleBody = shortenedArticle 
} else {
```

Reply:
that should make it stop throwing, but if you want to make sure that it's never gonna break, we gotta remove all custom components from markdown in order to be able to slice it. here's example rehypePlugin that removes all code blocks: https://github.com/skillrecordings/products/blob/main/apps/epic-react/src/utils/mdx.tsx#L34